config/config.go

```diff
@@ -333,6 +334,9 @@ type UIDefaults struct {
 // KialiFeatureFlags available from the CR
 type KialiFeatureFlags struct {
 	IstioInjectionAction bool       `yaml:"istio_injection_action,omitempty" json:"istioInjectionAction"`
+	IstioUpgradeAction bool       `yaml:"istio_upgrade_action,omitempty" json:"istioUpgradeAction"`
+	IstioCurrentRevision string       `yaml:"istio_current_revision,omitempty" json:"istioCurrentRevision"`
+	IstioUpgradeRevision string       `yaml:"istio_upgrade_revision,omitempty" json:"istioUpgradeRevision"`
```

Comment:
I'm not sure what these are (I assume these are related to the Istio canary deployments), but they don't appear to be appropriate for the Kiali Feature Flags. If this is related to istio canary deployment versions, consider putting these in `external_services.istio` along with the other istio-specific things (you'll see we already have things like IstiodDeploymentName which is also used mainly to support canary deployments)

Reply:
These configurations are for new Kiali UI Overviews page's feature to inject the canary version Istio injection label on namespace and trigger pods restart. Are quite similar to 'IstioInjectionAction' flag and coming from Kiali CR and used in kiali-ui.

Reply:
OK. I think we should consider renaming these - because I still don't understand what these are. When I see the name "IstioCurrentRevision" - that to me seems like the value of this setting will be something like "1.10" and when I see the name "IstioUpgradeRevision" - that to me seems like the value of this setting will be something like "1.11". And those don't seem to be related to just a Kiali UI function. That seems to be metadata about the Istio that is running. But I don't think my assumption is correct - based on your comment, these aren't version numbers, but instead are something that relates to injecting values into resources?

Should these be renamed to "IstioInjectCurrentVersion" and "IstioInjectUpgradeRevision"?